llvm/lib/CodeGen/SelectionDAG/SelectionDAGBuilder.cpp

```diff
@@ -3399,7 +3399,10 @@ void SelectionDAGBuilder::visitCallBr(const CallBrInst &I) {
     BasicBlock *Dest = I.getIndirectDest(i);
     MachineBasicBlock *Target = FuncInfo.getMBB(Dest);
     Target->setIsInlineAsmBrIndirectTarget();
-    Target->setMachineBlockAddressTaken();
+    // If there was a type of asm goto statement that was permitted to
```

Comment:
This comment makes sense in the context of this review. I'm not sure if I would understand it without.

I'd be tempted to remove it, relying on someone implementing a future extension to look at this patch to work out what to do. Alternatively it could just be a small rewording. Using "was" implies past tense to me, but I think this comment is about what to do if another type of asm goto is added, which would be in the future.